cmd/baton-trayai/main.go

```diff
@@ -42,13 +40,13 @@ func main() {
 	}
 }
 
-func getConnector(ctx context.Context, v *viper.Viper) (types.ConnectorServer, error) {
+func getConnector(ctx context.Context, tac *cfg.Trayai) (types.ConnectorServer, error) {
 	l := ctxzap.Extract(ctx)
-	if err := ValidateConfig(v); err != nil {
+	if err := field.Validate(cfg.Config, tac); err != nil {
```

Comment:
since there was nothing happening in `ValidateConfig`, I switched this to use `field.Validate` to make sure required fields are present.